src/go/rpk/pkg/cli/cmd/acl/create.go

```diff
@@ -0,0 +1,267 @@
+// Copyright 2021 Vectorized, Inc.
+//
+// Use of this software is governed by the Business Source License
+// included in the file licenses/BSL.md
+//
+// As of the Change Date specified in that file, in accordance with
+// the Business Source License, use of this software will be governed
+// by the Apache License, Version 2.0
+
+package acl
+
+import (
+	"context"
+	"fmt"
+	"strings"
+
+	"github.com/Shopify/sarama"
+	"github.com/pkg/errors"
+	log "github.com/sirupsen/logrus"
+	"github.com/spf13/cobra"
+	"golang.org/x/sync/errgroup"
+)
+
+func NewCreateACLsCommand(
+	admin func() (sarama.ClusterAdmin, error),
+) *cobra.Command {
+	var (
+		resource        string
+		resourceName    string
+		namePattern     string
+		operations      []string
+		allowPrincipals []string
+		allowHosts      []string
+		denyPrincipals  []string
+		denyHosts       []string
+	)
+	command := &cobra.Command{
+		Use:          "create",
+		Short:        "Create ACLs",
+		SilenceUsage: true,
+		Args: func(_ *cobra.Command, _ []string) error {
+			if len(allowPrincipals) == 0 &&
+				len(denyPrincipals) == 0 {
+				return fmt.Errorf(
+					"at least one of --%s or --%s must be set",
+					allowPrincipalFlag,
+					denyPrincipalFlag,
+				)
+			}
+
+			if resource == "cluster" {
+				resourceName = defaultClusterResourceName
+			}
```

Comment:
Maybe I missing some context, but why this code changes `cluster` resource to `kafka-cluster`? 

Redpanda doesn't allow `cluster`?

Reply:
Notice it doesn't change the resource, it just set the resource's name. For the `cluster` resource, the name will always be `kafka-cluster` in redpanda.

Reply:
You right, thanks.